oauth2-proxy/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -z ${OAUTH_CREDS_VAULT_PATH} ]; then
+  if [ -z ${VAULT_URL} ] || [ -z ${VAULT_AUTH_PARAM} ] || [ -z ${VAULT_AUTH_METHOD} ]; then
```

Comment:
I don't think you need the extra check for `VAULT_AUTH_METHOD` as you already set it in your Dockerfile ;)

Reply:
Well, you could still override it to an empty value in docker run 😛